### PR TITLE
Functional VerticalAccordion

### PR DESCRIPTION
**Pull request by bgptr**
This diff refactors `VerticalAccordion` to functional component using hooks and CSS modules.
Also, it includes automated tests.

Part of #2438 and #2439

### CODE REVIEW

File: app/components/shared/VericalAccordion/VerticalAccordion.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    (node) => {
+      if (node !== null && childHeight != node.clientHeight) {
+        setChildHeight(node.clientHeight);
+      }
+    },
+    [childHeight]
+  );
+
+  // "default" style when initializing the component
+  function getDefaultStyles() {
```
Comment (amassarwi) [inline app/components/shared/VericalAccordion/VerticalAccordion.js:28]: arrow func

File: app/components/shared/VericalAccordion/VerticalAccordion.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        style: {
+          height: 0,
+          opacity: 0
+        }
+      }
+    ];
+  }
+
+  // this returns the chosen style based on the passed props
+  function chosenStyles() {
```
Comment (amassarwi) [inline app/components/shared/VericalAccordion/VerticalAccordion.js:41]: arrow func

File: app/components/views/LNPage/PaymentsTab/PaymentsTab.module.css [hunk 3]
```diff
@@ -112,7 +112,7 @@
   grid-template-columns: 1fr 3fr;
 }
 
-.paymentAccordion.vertical-accordion-arrow {
+.paymentAccordion .vertical-accordion-arrow {
```
Comment (amassarwi) [inline app/components/views/LNPage/PaymentsTab/PaymentsTab.module.css:115]: Hrmm didn't test that, but it looks problematic the way it's now.

This should be in camel case instead like all other classes, but also I think this is redundant and it
has to effect as it used now in PaymentsTab/Page.jsx:

```
  arrowClassName="vertical-accordion-arrow"
  className={styles.paymentAccordion}
```

**Review by amassarwi — COMMENTED** [review]
Looks good overall, left some comments.

Also here, we should link to #2439.

**Comment by bgptr** [comment]
@amassarwi Thank you for your review. I've fixed them.

**Review by alexlyp — APPROVED** [review]

**Pull request merged**